### PR TITLE
MNT Add tests for multi-WYSIWYG config regressions

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Description

Tests for regressions for having multiple WYSIWYG configs.
Tests are for https://github.com/silverstripe/silverstripe-admin/issues/1512 and https://github.com/silverstripe/silverstripe-elemental/issues/1120

Needs all three of the below PRs to be merged for the CI to go green:
- https://github.com/silverstripe/silverstripe-admin/pull/1653
- https://github.com/silverstripe/silverstripe-asset-admin/pull/1435
- https://github.com/silverstripe/silverstripe-cms/pull/2916

## Manual testing steps

Load up an environment with the three linked PRs included, and run the new behat test.

## Issues

- https://github.com/silverstripe/silverstripe-elemental/issues/1120

## Pull request checklist

- [x] The target branch is correct
    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
- [x] All commits are relevant to the purpose of the PR (e.g. no debug statements, unrelated refactoring, or arbitrary linting)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
- [x] The commit messages follow our [commit message guidelines](https://docs.silverstripe.org/en/contributing/code/#commit-messages)
- [x] The PR follows our [contribution guidelines](https://docs.silverstripe.org/en/contributing/code/)
- [x] Code changes follow our [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/)
- [x] This change is covered with tests (or tests aren't necessary for this change)
- [ ] CI is green

### CODE REVIEW

File: tests/Behat/Extension/UniqueHtmlEditorConfigExtension.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorConfig;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
+
+/**
+ * @extends Extension<BaseElement>
```
Comment (GuySartorelli) [inline tests/Behat/Extension/UniqueHtmlEditorConfigExtension.php:13]: This adds no benefit but does no harm in 4.x.
In 5.x it tells your IDE what `$this->owner` and `$this->getOwner()` are.

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Seems like this should be in admin, not elemental?

**Comment by GuySartorelli** [comment]
Relies on elemental blocks - has to be in elemental.

Update: I have tried to reproduce this without elemental and haven't been able to do so.

**Review by emteknetnz — APPROVED** [review]

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Code is fine, CI is red

**Comment by GuySartorelli** [comment]
Rerunning CI - as per PR description it needed the others merged first before it could go green.

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**